### DISCUSSION TITLE
Neumorphism.Avalonia 11-prev4

### DISCUSSION THREAD

**Discussion by fomines** [flarive/Neumorphism.Avalonia#13] [General]
Hi!
We have a project based Avalonia 11-prev4. Do you have a version Neumorphism for Avalonia 11-prev4? 
Or maybe Neumorphism.Avalonia 11-prev5 normally integrated with  Avalonia 11-prev4?

**Comment by flarive** [comment]
Hi,
No there is no Neumorphism Avalonia integration done for preview4.
I haven't finished the integration with preview5, it's not working yet.
